cli/stream_command.go

```diff
@@ -2170,14 +2212,21 @@ func (c *streamCmd) prepareConfig(_ *fisk.ParseContext, requireSize bool) api.St
 		}
 	}
 
-	if c.repubSource != "" && c.repubDest != "" {
+	if c.repubSource != "" || c.repubDest != "" {
 		cfg.RePublish = &api.RePublish{
 			Source:      c.repubSource,
 			Destination: c.repubDest,
 			HeadersOnly: c.repubHeadersOnly,
 		}
 	}
 
+	if c.subjectTransformDest != "" {
+		cfg.SubjectTransform = &api.SubjectTransformConfig{
+			Source:      c.subjectTransformSource,
```

Comment:
Do we support empty string source? Seems we should check both are not empty. I know you check it in other places to detect both arent empty but seems kind of action at a distance while it would more maintainable if we check things right here instead

Reply:
So I made the server transform code tolerant to "" source being treated like ">" source (anywhere transform is used), obviously doesn't make sense to have an empty transform so that's an error returned by the server. So right now it lets you just specify a destination when creating the stream.

That said I'd rather people did always use ">" rather than empty, which I did later by forcing the user to specify both source and dest arguments. And also I made info display a ">" if the config source is "".

It related to the requirement above of requiring both source and dest for the republish.

I can update the if to check both source && dest if you want?

Reply:
Yeah I am not in favor of the magical empty here, rather people are explicit.

Reply:
Done: changed to && on both source and dest